DOL.WHD.Section14c.Web/src/v4/ui-library.component.html

```diff
@@ -0,0 +1,234 @@
+        <button class="usa-button-secondary">B</button>
+      </td>
+    </tr>
+    <tr class="expanded-row greyed" ng-repeat-end="">
+      <td></td>
+      <td colspan="3">
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<h2 class="mb3">Misc</h2>
+
+<div class="mb5">
+  <nav role="navigation" class="main-nav-bar collapsed">
+    <ul>
+      <li class="nav-button active" data-sectionid="assurances">Assurances</li>
+      <li class="nav-button" data-sectionid="app-info">Application Info</li>
+      <li class="nav-button" data-sectionid="employer">Employer</li>
+      <li class="nav-button" data-sectionid="wage-data">Wage Data</li>
+      <li class="nav-button" data-sectionid="work-sites">Work Sites & Employees</li>
+      <li class="nav-button" data-sectionid="wioa">WIOA</li>
+      <li class="nav-button ng-hide" data-sectionid="review">Review & Submit</li>
+    </ul>
+  </nav>
+</div>
+
+<div class="mb5">
+  <div id="wagedata_tab_box" class="form-tabbed-section">
+    <div class="form-tab active">
+      <div class="circle">1</div> Hourly
+    </div>
+    <div class="form-tab ">
+      <div class="circle">2</div> Piece Rate
+    </div>
+  </div>
+</div>
+
+<div class="mb5">
+  <div class="dol-applicant-meta">
+    <div class="dol-feedback-save">
+      <span class="dol-feedback-save-success ng-binding">
+        Saved 09/11/2017, 11:53 AM
+      </span>
+    </div>
+    <div class="dol-applicant-ein">
+      Applicant EIN:
+      <span class="ng-binding">12-1233333</span>
+    </div>
+  </div>
+  <div class="dol-form-meta">
+    <div class="usa-content">
+      <p>Application for Authority to Employ Workers with Disabilities at Subminimum Wages</p>
+      <p class="ng-hide">Applicant EIN:
+        <span class="ng-binding">XX-XXXXXXX</span>
+      </p>
+    </div>
+    <div class="dol-form-omb" ng-show="!admin">
+      <p>OMB NO: 1235-0001</p>
+      <p>REV XX/20XX</p>
+    </div>
+  </div>
+</div>
+
+<div class="mb5">
+  <div class="main-content form-content">
+    <div class="usa-alert usa-alert-error" role="alert">
+      <div class="usa-alert-body">
+        <p class="usa-alert-text">Please review this page. There are errors or missing information that we need to collect before you can submit the form.</p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="mb5">
+  <div class="reviewbar">
+    <div class="reviewbar-container">
+      <div class="reviewbar-bar"></div>
+      <div class="reviewbar-circle"></div>
+    </div>
+  </div>
+  <div class="reviewbar error">
+    <div class="reviewbar-container">
+      <div class="reviewbar-bar"></div>
+      <div class="reviewbar-circle"></div>
+    </div>
+  </div>
+</div>
+
+<div class="mb5">
+  <div class="dol-panel-login">
+    <h2>Log in</h2>
+    <p>...</p>
+  </div>
+</div>
+
+<div class="mb5">
+  <div class="dol-panel-info mw-300p">
+    <h3>Questions?</h3>
+    <p>Contact the <a href="#">Wage and Hour office closest to you.</a></p>
+  </div>
+</div>
```

Comment:
I was thinking this was going to be all of the Angular components, so that they would be rendered by Angular and everything, not the template HTML copied from them.  The reason is twofold - first, it ensures that we're testing what the actual Angular produces, and second, it would keep this page in sync with the active production versions of the components.

Ultimately this is a question for @mmurthydol - is this what you intended?

Reply:
i was thinking that as well at first too, but a lot of these components rely on some sort of state that would difficult to fully port over / manage from this one page, so I opted to simply extract the relevant html bits. This does create some potential two-sources-of-truth issues but my thinking was that would be alright since the purpose of this page (based on my understanding) is really to flag big accessibility bugs to be fixed (and once that's done this page can be ignored / disposed of) vs a longer living component style guide

Reply:
Ah, that makes sense.  👍